raiden/network/transport/matrix.py

```diff
@@ -782,21 +792,25 @@ def _handle_message(self, room, event) -> bool:
         # rooms we created and invited user, or were invited specifically by them
         room_ids = self._get_room_ids_for_address(peer_address)
 
-        if room.room_id not in room_ids:
+        # TODO: Remove clause after `and` and check if things still don't hang
```

Comment:
Is this TODO still valid? Sounds brittle. Isn't it possible to have tests for all of these?

Reply:
Yes, that is new and it's valid. It's related to the one below.

Both of those need to be investigated but since we want to start testing and it's non-critical I'd say lets merge this now and handle those afterwards.